src/sentry/integrations/base.py

```diff
@@ -95,6 +95,7 @@ class IntegrationFeatures(Enum):
     SERVERLESS = "serverless"
     TICKET_RULES = "ticket-rules"
     STACKTRACE_LINK = "stacktrace-link"
+    CODEOWNERS = "codeowners"
```

Comment:
if you want this to be able to align with the feature gate it needs to be named the same so `import-codeowners` but actually now that i think about it, the feature gate should have been `integrations-import-codeowners`...